packages/flutter_adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -26,14 +26,27 @@ class MyApp extends StatelessWidget {
 /// [AdaptiveLayout].
 class MyHomePage extends StatefulWidget {
   /// Creates a const [MyHomePage].
-  const MyHomePage({super.key});
+  const MyHomePage({super.key, this.transTime = 1000});
+
+  /// Declare transition duration.
+  final int transTime;
```

Comment:
This should probably be `transitionDuration`. Our coding guidelines say to use full names where possible.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations